splunk-quarkus/clowdapp.yaml

```diff
@@ -101,8 +96,6 @@ parameters:
     value: >-
       -Djava.util.logging.manager=org.jboss.logmanager.LogManager
       -Dquarkus.log.file.enable=false
-  - name: KAFKA_INGRESS_TOPIC
-    description: Kafka topic that the consumer takes messages from
     value: platform.notifications.tocamel
```

Comment:
You missed to remove this as well. It belonged to the removed param.

Reply:
oops :-)